sourcemash/frontend/static/logotext.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
SVG scales and doesn't pixelate. PNG does.

And you can do that or email me the files. Just let me know if you make changes on the branch so I pull them.